src/pages/ApartmentShow.js

```diff
@@ -1,13 +1,60 @@
-import React from 'react';
 
+import React from 'react';
+import { Card, CardTitle, CardSubtitle, CardBody, CardText, CardLink, Button } from 'reactstrap';
+import { useParams, Link, NavLink } from 'react-router-dom';
 import '../styling/ApartmentShow.css';
+import ApartmentDelete from './ApartmentDelete';
+
+const ApartmentShow = ({ apartments, deleteApartment, logged_in, currentUser }) => {
+  const { id } = useParams();
+  let currentApt = apartments?.find((apt) => apt.id === +id);
+
+  const handleDelete = () => {
+    deleteApartment(currentApt.id);
+  };
+
+  if (!currentApt) {
+    return <div>Loading...</div>; // Add loading state or handle the case when currentApt is undefined
```

Comment:
This is a good comment to keep just in case. Just until you get a bit more experience and feel like you no longer need it